test/Field/golden/03-issue-76.golden

```diff
@@ -0,0 +1,2 @@
+ext x_2;
+let y_3 = if (0 == x_2) then 0 else 0;
```

Comment:
Ok, so we can conclude that #76 is fixed.

Reply:
It looks like it.  I would keep it as a regression test though.

Reply:
Sure.